src/pages/Cart.jsx

```diff
@@ -1,13 +1,61 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 
 class Cart extends Component {
+  constructor() {
+    super();
+    this.state = {
+      count: 0,
+    };
+    this.increment = this.increment.bind(this);
+    this.decrement = this.decrement.bind(this);
+  }
+
+  increment() {
+    const { count } = this.state;
+    this.setState({
+      count: count + 1,
+    });
+  }
+
+  decrement() {
+    const { count } = this.state;
+    if (count >= 0) {
```

Comment:
O que você acha de alterar o `if(count >= 0) ` por
```suggestion
    if (count > 1) {
```